web_ui/src/routes/auth/auth.layout.tsx

```diff
@@ -68,9 +68,15 @@ export const AuthenticationLayout = (): JSX.Element => {
         return <Outlet />;
     }
 
-    if (auth.isLoading || auth.activeNavigator || loginQuery.isPending) {
+    if (auth.isLoading || auth.activeNavigator) {
         return <IntelBrandedLoading />;
     }
 
-    return <>{auth?.error || loginQuery.isError ? <LoginErrorScreen /> : <Outlet />}</>;
+    if (auth.error) return <LoginErrorScreen />;
```

Comment:
In case of auth.error`Error: No matching state found in storage` loginQuery would be disabled and in `isPending` state. So we need to handle auth error before query status check